.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -28,4 +28,5 @@ Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-
   - [ ] (Package updates) Added a release notes entry if the change is major or breaking
   - [ ] (Module updates) Added a release notes entry if the change is significant
   - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
+  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
```

Comment:
> I think that it's not sufficient: that doc is not structured as a checklist, but PR needs one.

Also this is duplicated with the point one level up. Why not convert that into a checklist instead of duplicating the info that is linked there?

Reply:
If I understood you correctly, then yeah — restructuring doc you've mentioned may prove useful.

Solution I propose tries to mitigate a pain point associated with adding "release notes" section.
As I see it, checklist itself is not designed to be a full documentation for how to contribute to nixpkgs -- it is only here to address critical-ish stuff people should really be reminded of checking before making PRs. 

Even if they don't check those boxes, they would at least see that there is such thing as "generating release notes if release notes were added".

And "single source of truth" as good as it sounds wouldn't be read by most of our conributors :(